modules/fscloud/variables.tf

```diff
@@ -117,5 +117,18 @@ variable "verify_worker_network_readiness" {
   default     = true
 }
 
-
+variable "addons" {
+  type = object({
+    alb-oauth-proxy           = optional(string)
+    debug-tool                = optional(string)
+    image-key-synchronizer    = optional(string)
+    istio                     = optional(string)
+    openshift-data-foundation = optional(string)
+    static-route              = optional(string)
+    cluster-autoscaler        = optional(string)
+    vpc-block-csi-driver      = optional(string)
+  })
+  description = "List of all addons supported by the ocp cluster. For information on supported versions, see https://cloud.ibm.com/docs/openshift?topic=openshift-supported-cluster-addon-versions"
```

Comment:
Can you update the description to this:

> Map of OCP cluster add-on versions to install (NOTE: The 'vpc-block-csi-driver' add-on is installed by default for VPC clusters, however you can explicitly specify it here if you wish to choose a later version than the default one). For full list of all supported add-ons and versions, see https://cloud.ibm.com/docs/containers?topic=containers-supported-cluster-addon-versions

Also update it in the main module too please